docs/setup.md

```diff
@@ -66,6 +70,24 @@ Usage details:
 | `-r / --ref` | Path to reference fasta. Mutually exclusive with -a|
 | `-a / --acc` | NCBI accession of reference. Mutually exclusive with -r|
 
+#### Handling samples with more than one pair of reads
+
+In order to specify samples that were sequenced multiple times in your sample sheet, you must:
+1. Create a duplicate row for each unit of sequencing
+2. Ensure the `BioSample` value is the same across all rows for the sample.
+3. Give each row a unique `Run` value. This allows snpArcher to collect all alignments for a `BioSample`.
```

Comment:
i think instead of alignments write "read pairs". maybe after the period add "All runs for a sample will be mapped separately to the genome and subsequently merged"